daemon/src/maker_cfd.rs

```diff
@@ -81,7 +78,7 @@ impl Actor {
         let mut conn = db.acquire().await?;
 
         // populate the CFD feed with existing CFDs
-        cfd_feed.update(&mut conn).await?;
+        cfd_feed_actor_inbox.send(load_all_cfds(&mut conn).await?)?;
```

Comment:
same here, since restart we have a list of `cfds` in here, and the constructor should not have side effects in changing cfd state, so we could just use the list we get and send it out.